app/models/booking.rb

```diff
@@ -1,6 +1,8 @@
 class Booking < ActiveRecord::Base
 	belongs_to :flight
-  	has_many :passengers
+  	has_many :passengers, dependent: :destroy
 
-  	accepts_nested_attributes_for :passengers
+  	accepts_nested_attributes_for :passengers, reject_if: :all_blank, allow_destroy: true
```

Comment:
Line is too long. [88/80]<br>Inconsistent indentation detected.<br>Tab detected.